src/artist.coffee

```diff
@@ -625,7 +625,7 @@ class Artist
         fingerings = @makeFingering(annotations[i])
         if fingerings?
           try
-            (note.addModifier(fingering.num, fingering.modifier) for fingering in fingerings)
+            (note.addModifier(fingering.modifier, fingering.num) for fingering in fingerings)
```

Comment:
Possible bug fix? Or did an old version of VexFlow have the arguments reversed? Anyways, it was throwing an exception so I switched the two arguments.